### PR TITLE
Create کتاب‌های داستانی.md

### PR DESCRIPTION
**Pull request by HosnaSltn**

### CODE REVIEW

File: docs/کلاب‌ها/بوک کلاب/کتاب‌های داستانی.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+![image]()
+
+
+# انسان‌ها
```
Comment (AminMasoudi) [inline docs/کلاب‌ها/بوک کلاب/کتاب‌های داستانی.md:4]:
```suggestion

# کتاب های داستانی
## انسان‌ها
```

**Review by AminMasoudi — CHANGES_REQUESTED** [review]

File: docs/کلاب‌ها/بوک کلاب/کتاب‌های داستانی.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+**نویسنده: مت هیگ**
+
+
+‌
+
+
+🔹  **ژانر:** داستانی، طنز، علمی تخیلی
+
+
+🔹  **خلاصه:** وقتی ریاضیدانی بزرگترین مسئله‌ی حل نشده‌ی ریاضی را اثبات می‌کند، موجودی فضایی مأمور می‌شود بدن او را تسخیر کرده و تمامی آثار آن را پاک کند تا انسان‌ها که در دیدش موجوداتی منفور هستند از آن سوءاستفاده نکنند. اما زندگی در زمین تجربه‌ای عجیب و چالش برانگیز است که او انتظارش را ندارد...
```
Comment (HosnaSltn) [inline docs/کلاب‌ها/بوک کلاب/کتاب‌های داستانی.md:18]:
```suggestion

```

**Review by HosnaSltn — COMMENTED** [review]
ok

File: docs/کلاب‌ها/بوک کلاب/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+![image]()
```
Comment (AminMasoudi) [inline docs/کلاب‌ها/بوک کلاب/index.md:1]: i've assume that You've forgotten to upload the image of the club.

**Review by AminMasoudi — APPROVED** [review]
Nice job

**Pull request merged**